packages/pds-ember/app/styles/pds/theme/index.scss

```diff
@@ -1,4 +1,199 @@
+//   - Spacing (margin, padding, etc.)
+//   - Font Size (headings, body copy, subtext, supertext, etc.)
+$size--2xs: scale.$sm-3; // 4px  (0.25rem)
+$size--xs:  scale.$sm-2; // 8px  (0.5rem)
+$size--sm:  scale.$sm-1; // 12px (0.75rem)
+$size--md:  scale.$base; // 16px (1rem)
+$size--lg:  scale.$lg-2; // 24px (1.5rem)
+$size--xl:  scale.$lg-5; // 36px (2.25rem)
+$size--2xl: scale.$lg-7; // 48px (3rem)
+
+
+// RESPONSIVE BREAKPOINTS
+// ----------------------
+// Sass Only! (DO NOT allow client configuration)
+//
+// Even if we did allow client a11y configuration,
+// CSS custom properties can't be used in @media
+// queries.  Media Queries Level 5 spec will enable
+// this, but it's not yet available in any browser.
+// Tablets & Low-res PCs
+$media--sm: resolutions.$svga-width !default;
+// High-res PCs
+$media--md: resolutions.$hd-width !default;
+// Ultra High-res PCs
+$media--lg: resolutions.$fhd-width !default;
+
+
+
+// CORE TYPOGRAPHY
+// ---------------
+$fontFamily: font.$family-system-sans;
+$fontFamily--code: font.$family-system-mono;
+$fontSize: $size--md; // 16px (1rem)
+$fontSize--lg: scale.$lg--1; // 20px
+$fontSize--sm: scale.$sm--1; // 12px
+$fontWeight: font.weight(Regular);
+
+
+// HEADINGS
+// --------
+$heading-fontSize--0: scale.$lg--5; // 36px;
+$heading-fontSize--1: scale.$lg--3; // 28px
+$heading-fontSize--2: $fontSize--lg;
+$heading-fontSize--3: scale.$base; // 16px
+$heading-fontSize--4: $_size14;
+$heading-lineHeight: $lineHeight--dense;
+$heading-fontWeight: font.weight(Bold);
+
+// UI ELEMENTS
+// -----------
+$ui-fontSize: $_size14;
+$ui-fontSize--sm: $fontSize--sm; // 12px
+$ui-fontSize--lg: scale.$base; // 16px (1rem)
+$ui-fontWeight: font.weight(Medium);
+$ui-lineHeight: $lineHeight--dense;
+
+// LABELS
+// ------
+$label-fontSize: $_size14;
+$label-fontWeight: font.weight(Bold);
+$label-lineHeight: $lineHeight--dense;
+
+// INLINE CODE
+// -----------
+$inlineCode-backgroundColor: color.$ui-gray-050;
+$inlineCode-borderColor: color.$ui-gray-200;
+$inlineCode-borderRadius: $borderRadius;
+$inlineCode-color: color.$magenta-500;
+$inlineCode-font: inherit;
+$inlineCode-fontFamily: font.$family-system-mono;
+$inlineCode-lineHeight: inherit;
+$inlineCode-padding: 0 $size--2xs;
+
+// CODE BLOCK
+// ----------
+$codeBlock-fontFamily: font.$family-system-mono;
+$codeBlock-fontSize: $_size14;
+$codeBlock-fontWeight: font.weight(Medium);
+$codeBlock-lineHeight: $lineHeight;
+
+// CONTROLS/INPUTS (inline)
+// ------------------------
+$control-height: scale.$lg--4; // ~32px
+$control-fontSize: $ui-fontSize;
+
+
+// @use "pds/theme";
+// .foo {
+//   box-shadow: theme.boxShadow(low, focus);
+// }
+@function boxShadow($names...) {
+  @return Shadow.build($names...);
+}
+
+@mixin styles {
+  // Global Theme
+  :root {
+    // default to "light mode"
+    @include Light.styles;
+    // switch to "dark mode" (if preferred)
+    @include Prefers.darkMode {
+      @include Dark.styles;
```

Comment:
Currently only spits out a CSS comment to vet out behavior of the theming framework.  Once we implement dark mode, this will override CSS custom properties defined by `Light.styles` above.